gems/quilt_rails/test/dummy/app/assets/config/manifest.js

```diff
@@ -0,0 +1,4 @@
+//= link_tree ../images
```

Comment:
do we need this? We don't actually serve anything atm

Reply:
Yeah, I am leaning toward deleting all the testing stuff.